src/js/views/maps/ViewfinderView.js

```diff
@@ -0,0 +1,154 @@
+"use strict";
+
+define([
+  "backbone",
+  "text!templates/maps/viewfinder.html",
+], (
+  Backbone,
+  Template,
+) => {
+  /**
+   * @class ViewfinderView
+   * @classdesc The ViewfinderView allows a user to search for a latitude and longitude in the map view.
+   * @classcategory Views/Maps
+   * @name ViewfinderView
+   * @extends Backbone.View
+   * @screenshot views/maps/ViewfinderView.png
+   * @since 2.27.1
+   * @constructs ViewfinderView
+   */
+  var ViewfinderView = Backbone.View.extend({
+    /**
+     * The type of View this is
+     * @type {string}
+     */
+    type: "ViewfinderView",
+
+    /**
+     * The HTML classes to use for this view's element
+     * @type {string}
+     */
+    className: classNames.baseClass,
+
+    /**
+    * The events this view will listen to and the associated function to call.
+    * @type {Object}
+    */
+    events: {
+      [`change .${classNames.input}`]: 'valueChange',
+      [`click .${classNames.button}`]: 'search',
+      [`keyup .${classNames.input}`]: 'keyup',
+    },
+
+    /** 
+     * Values meant to be used by the rendered HTML template.
+     */
+    templateVars: {
+      errorMessage: "",
+      // Track the input value across re-renders.
+      inputValue: "",
+      placeholder: "Search by latitude and longitude",
```

Comment:
Consider showing an example lat and long instead

Reply:
As this feature gets more functionality (like geocoder integration) I think it will be helpful to have the examples of possible search queries live in the "error" messaging.